### PR TITLE
Update old release notes for readthedocs

### PR DESCRIPTION
**Pull request by aeng**
Update release notes for readthedocs

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+
+-----------------------
+
+## 3.0.0
+<h5>New Features</h5>
+
+* [980659](https://bugzilla.redhat.com/show_bug.cgi?id=980659) - TMX import/export
+    * Imported TMX shown in translation memory search results
+    * Allow users to export translations to TMX (from Project/Version pages)
+    * Allow admins to export **all** project translations to TMX (from Projects page)
```
Comment (djansen-redhat) [inline docs/release-notes.md]: Will this make the "all" unnecessarily bold?

File: docs/release-notes.md [hunk 2]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+* [786630](https://bugzilla.redhat.com/show_bug.cgi?id=786630) - Shortcut Alt+G causes editor to lose focus
+* [870876](https://bugzilla.redhat.com/show_bug.cgi?id=870876) - PO download for non-PO projects cannot be uploaded
+* [846314](https://bugzilla.redhat.com/show_bug.cgi?id=846314) - Show validation state in doc list and link to error-filter view in editor
+* [844553](https://bugzilla.redhat.com/show_bug.cgi?id=844553) - Notification of an entry should have a link that go to the entry
+* [727826](https://bugzilla.redhat.com/show_bug.cgi?id=727826) - Order Projects alphabetically
+* [917911](https://bugzilla.redhat.com/show_bug.cgi?id=917911) - Keep "Validation Warnings: n" displayed even when moving focus to different pages
+* [910637](https://bugzilla.redhat.com/show_bug.cgi?id=910637) - Keep "Validation Warnings: n" displayed even when moving focus to different entry
+* [767055](https://bugzilla.redhat.com/show_bug.cgi?id=767055) - Error when pull as XLIFF file format: Underlying stream encoding 'ASCII' and input paramter for writeStartDocument() method 'utf-8' do not match
```
Comment (djansen-redhat) [inline docs/release-notes.md]: "paramter"

File: docs/release-notes.md [hunk 3]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+
+* [917895](https://bugzilla.redhat.com/show_bug.cgi?id=917895) - Validation rules should be enabled by default
+* [917897](https://bugzilla.redhat.com/show_bug.cgi?id=917897) - AlreadyClosedException when new document uploaded and translated
+* [807100](https://bugzilla.redhat.com/show_bug.cgi?id=807100) - Removing admin role doesn't take effect for Kerberos authentication
+
+-----------------------
+
+## 2.2.1
+
+* [915130](https://bugzilla.redhat.com/show_bug.cgi?id=915130) - Unexpected error when clicking "resend activation email" or "update emaill address"
```
Comment (djansen-redhat) [inline docs/release-notes.md]: "emaill"

File: docs/release-notes.md [hunk 4]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+* Add attention shortcut to copy from source: Alt+X,G
+* File upload
+* Move raw document storage to file system
+
+-----------------------
+
+## 2.3.2
+
+* [958350](https://bugzilla.redhat.com/show_bug.cgi?id=958350) - Concurrent save on different row causes incorrect validation warnings in current row
+* [959060](https://bugzilla.redhat.com/show_bug.cgi?id=959060) - Generated zanata config file contains invalid project type
```
Comment (djansen-redhat) [inline docs/release-notes.md]: lower case Z

File: docs/release-notes.md [hunk 5]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+* [887052](https://bugzilla.redhat.com/show_bug.cgi?id=887052) - Source and Target search in editor fails when the search term includes an apostrophe (')
+* [888150](https://bugzilla.redhat.com/show_bug.cgi?id=888150) - Case sensitive search should return case sensitive results
+* [877228](https://bugzilla.redhat.com/show_bug.cgi?id=877228) - Clearing the search field in workspace should keep position at last selected message
+* [880444](https://bugzilla.redhat.com/show_bug.cgi?id=880444) - enable spell check in code mirror editor for Firefox
+* [880879](https://bugzilla.redhat.com/show_bug.cgi?id=880879) - Undo button causing repeated save failures and other weirdness
+* [884402](https://bugzilla.redhat.com/show_bug.cgi?id=884402) - Entry should NOT move unless it is absolutely needed
+* [884502](https://bugzilla.redhat.com/show_bug.cgi?id=884502) - navigation breaks in filter mode after saved status not included in filter view
+* [887717](https://bugzilla.redhat.com/show_bug.cgi?id=887717) - enable 'Enter' key saves immediately will make pager input disfunctional
```
Comment (djansen-redhat) [inline docs/release-notes.md]: Should be "dysfunctional"

File: docs/release-notes.md [hunk 6]
```diff
@@ -257,3 +252,548 @@ Zanata now requires JMS to be configured in standalone.xml in order to queue up
...
+* [803923](https://bugzilla.redhat.com/show_bug.cgi?id=803923) - email should be able to corrected during register validation
+* [829565](https://bugzilla.redhat.com/show_bug.cgi?id=829565) - Kerberos activation link in email gets 404 page not found
+* [872039](https://bugzilla.redhat.com/show_bug.cgi?id=872039) - Escaping with single-quote (a.k.a. Apostrophes ') character in MessageFormat strings can cause confusing validation warnings
+* [886711](https://bugzilla.redhat.com/show_bug.cgi?id=886711) - Error when using pull for project type raw when the document name does not include a type extension
+* [831056](https://bugzilla.redhat.com/show_bug.cgi?id=831056) - Option for highlight only the search terms
+* [785046](https://bugzilla.redhat.com/show_bug.cgi?id=785046) - Limit source string length in properties file
+* [846643](https://bugzilla.redhat.com/show_bug.cgi?id=846643) - Shorten the navigation sequence to open a document in the Editor
+* [884386](https://bugzilla.redhat.com/show_bug.cgi?id=884386) - email validation link should be invalid after user validiate the email, or user request another validation
```
Comment (djansen-redhat) [inline docs/release-notes.md]: s/validiate/validate/

**Comment by djansen-redhat** [comment]
Feel free (or not) to fix the errors in spelling from BZ.  Otherwise, :+1:

File: docs/release-notes.md (hunk 1 above)
Comment (aeng) [inline docs/release-notes.md]: nope. checked locally.

**Comment by djansen-redhat** [comment]
:+1:

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/9fd2486b7932d983c0ce64617658816119a2c42e

**Pull request closed without merging**